waterbox/dosbox/bizhawk.cpp

```diff
@@ -210,7 +210,7 @@ ECL_EXPORT void FrameAdvance(MyFrameInfo* f)
 	if (f->driveActions.insertCDROM >= 0)
 	{
 		printf("Swapping to CDROM: %d\n", f->driveActions.insertCDROM);
-		swapInDrive(3, f->driveActions.insertFloppyDisk + 1); // 3 is D:
+		swapInDrive(3, f->driveActions.insertCDROM + 1); // 3 is D:

```

Comment:
Is it fixed which letter each type of medium is inserted into?

Reply:
Looks like yes. My final wish is to have an enum for those letters so we don't have to pass magic numbers coupled with comments. A `#define` is acceptable too.